### PR TITLE
tools: scripts: Create BOOT.BIN and export them

### PR DESCRIPTION
**Pull request by AAndrisa**

### CODE REVIEW

File: tools/scripts/build_projects.py [hunk 1]
```diff
@@ -195,19 +195,26 @@ def __init__(self, project_dir, platform, flags, build_name, hardware,
...
 		if hardware != '':
 			short_build_dir = short_build_dir + '_' + hardware
 			self._binary = "%s_%s_%s_%s.elf" % (
 				self.project, platform, build_name, hardware)
+			self.boot_dir = "%s_%s_%s_%s" % (
+				self.project, platform, build_name, hardware)
 		self.build_dir = os.path.join(self.builds_dir, short_build_dir)
 		self.binary = os.path.join(self.build_dir, self._binary)
 		self.export_file = os.path.join(self.build_dir, self.binary)
+
```
Comment (amiclaus) [inline tools/scripts/build_projects.py:208]: unrelated change.

File: tools/scripts/xilinx.mk [hunk 2]
```diff
@@ -85,6 +98,7 @@ LDFLAGS += -mcpu=cortex-r5						\
            -mfloat-abi=hard						\
 	   -mfpu=vfpv3-d16						\
 	   -DARMR5
+
```
Comment (amiclaus) [inline tools/scripts/xilinx.mk:101]: ditto

**Review by amiclaus — COMMENTED** [review]
please sign the CLA. some initial nitpicks on my side

**Review by amiclaus — COMMENTED** [review]
please try to provide in the commit title th paths/semi-paths to the files/folders in which you did changes.
Example:
`tools: scritps: create boot.bin and export`

**Review by amiclaus — COMMENTED** [review]
a general rule: after each push force please provide a changelog stating the changes that you made (usually updates related to the review).

Multiple xilinx projects builds are failing:
```[ERROR]  : Cannot read file - `tmp/output/hw0/export/hw0/sw/hw0/boot/fsbl.elf```

**Review by amiclaus — APPROVED** [review]

**Review by buha — APPROVED** [review]

**Comment by buha** [comment]
I'm also seeing these errors, they need to be fixed:
`[ERROR]  : Cannot read file - /home/azure/Desktop/builds/build_xilinx_ad4630_fmc_zed/tmp/output/hw0/export/hw0/sw/hw0/boot/fsbl.elf`

**Comment by buha** [comment]
Errors were fixed, they were caused by the build environment and not by these modifications per se.
